code/game/objects/items/circuitboards/machines/machine_circuitboards.dm

```diff
@@ -159,6 +160,53 @@
 	req_components = list(/datum/stock_part/capacitor = 1)
 	needs_anchored = FALSE
 
+/obj/item/circuitboard/machine/modular_shield_generator
+	name = "Modular Shield Generator"
+	greyscale_colors = CIRCUIT_COLOR_ENGINEERING
+	build_path = /obj/machinery/modular_shield_generator
+	req_components = list(
+		/datum/stock_part/servo = 1,
+		/datum/stock_part/micro_laser = 1,
+		/datum/stock_part/capacitor = 1,
+		/obj/item/stack/sheet/plasteel = 3
+		)
```

Comment:
```suggestion
		/obj/item/stack/sheet/plasteel = 3,
	)
```

Reply:
> some stuff slipped you by, i just replied to any un-addressed suggestions with "bump"

thanks its getting hard keeping track of these ontop of the ones i left open to see if you were happy with them or not.